capstone-app/src/app/modules/photos/components/photos-section/photos-section.component.ts

```diff
@@ -71,15 +89,16 @@ export class PhotosSectionComponent implements OnChanges {
    * 2) There are bad photos which cannot be fetched so the new limit is less than max
    */
   updateLimit(newLimit: number, wasRemoved: number) {
-    if (this.limitControl.value === this.maxPhotos && this.maxPhotos < newLimit) {
+    if (
+      this.limitControl.value === this.maxPhotos &&
+      this.maxPhotos < newLimit
+    ) {
```

Comment:
nit/optional: if (this.limitControl.value === this.maxPhotos &&
   this.maxPhotos < newLimit) {

Reply:
Thanks. My linter sometimes does weird formatting